Cooper/ClientApp/src.v2/app/layouts/profile/profile.layout.ts

```diff
@@ -118,4 +121,69 @@ export class ProfileLayoutComponent implements OnInit, OnDestroy {
       this.userService.updateUserInfo(updatedUser);
     }
   }
+
+  public subscribe(userId: number): void {
+    this.usersSocialConnectionsService.subscribe(userId).subscribe(
```

Comment:
why two subscribe and not clear what it is
too much functions with name subscribe